Snakefile

```diff
@@ -205,16 +205,16 @@ rule cc_single:
         --hvgs {params.hvgs} --organism {params.organism}
         """
 
-rule scale_lisi:
```

Comment:
If you remove this rule, you will also need to change `all:` to accept the output from the `metrics:` rule as output.

Reply:
Change `all:` rule in commit (a37f239 )[https://github.com/theislab/scib/pull/137/commits/a37f239ca98a282f66a660bc20fabb9c4e2199e9]

Reply:
The change in all is the essential change for snakemake :+1: . The rest is just cleaning up.